cs-onprem-tenant-config.sh

```diff
@@ -1,27 +1,8 @@
 #!/bin/bash
 configMapCustomHostname="cs-onprem-tenant-config"
 csNamespace=""
-custom_hostname=""
-topology=""
 map_to_common_service_namespace=""
-
-configmap_yaml=$(oc get configmap common-service-maps -n kube-public -ojsonpath='{.data.common-service-maps\.yaml}')
-temp_yaml_file=$(mktemp)
-echo "$configmap_yaml" > "$temp_yaml_file"
-count=$(cat $temp_yaml_file |grep -A2 'requested-from-namespace:' | sed -n '/^  - /p' |wc -l)
-if [ "$count" -eq 2 ]; then
-    # Fetch the value of the "map-to-common-service-namespace" property
-    map_to_common_service_namespace=$(cat $temp_yaml_file |grep 'map-to-common-service-namespace:' | awk '{print $3}')
-    topology="SOD"
-    csNamespace=$(cat $temp_yaml_file |grep -A2 'requested-from-namespace:' | sed -n '/^  - /p' |grep -v $map_to_common_service_namespace|awk '{print $2}')
-    #echo "The value of 'map-to-common-service-namespace' is: $map_to_common_service_namespace and control namespace is $csNamespace"
-else
-    map_to_common_service_namespace=$(cat $temp_yaml_file |grep 'map-to-common-service-namespace:' | awk '{print $3}')
-    topology="simple"
-    csNamespace=$map_to_common_service_namespace
-fi
-
-echo "This cluster is configured with $topology topology"
+custom_hostname=""
 wlp_client_id=$(oc get secret -n $map_to_common_service_namespace platform-oidc-credentials -o jsonpath='{.data.WLP_CLIENT_ID}'|base64 -d)
```

Comment:
Hi @DaniyalMustafa 
I am wondering that should we initialize the value for `map_to_common_service_namespace` and `csNamespace`?

Reply:
Got it, thank you @DaniyalMustafa 

Just check the documentation, yes we are asking user to initialize the values